### PR TITLE
[5.6] Symfony 4 Upgrade

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -28,15 +28,15 @@
...
-        "tijsverkoyen/css-to-inline-styles": "~2.2",
+        "symfony/console": "~4.0",
+        "symfony/debug": "~4.0",
+        "symfony/finder": "~4.0",
+        "symfony/http-foundation": "~4.0",
+        "symfony/http-kernel": "~4.0",
+        "symfony/process": "~4.0",
+        "symfony/routing": "~4.0",
+        "symfony/var-dumper": "~4.0",
+        "tijsverkoyen/css-to-inline-styles": "dev-master",
```
Comment (GrahamCampbell) [inline composer.json]: Waiting on new tagged release here.

File: src/Illuminate/Mail/composer.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
         "illuminate/support": "5.6.*",
         "psr/log": "~1.0",
         "swiftmailer/swiftmailer": "~6.0",
-        "tijsverkoyen/css-to-inline-styles": "~2.2"
+        "tijsverkoyen/css-to-inline-styles": "dev-master"
```
Comment (GrahamCampbell) [inline src/Illuminate/Mail/composer.json]: And here.

Comment (alexbowers) [reply src/Illuminate/Mail/composer.json]: @GrahamCampbell This has been tagged 27 days ago (2.2.1) is it worth changing it from dev-master. Seems that laravel shouldn't really rely on a branch as a dependency.

Comment (GrahamCampbell) [reply src/Illuminate/Mail/composer.json]: Yes, this was updated 27 days ago.

**Pull request merged**